### PR TITLE
Update env config staging

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
Remove environment-config.xml #3889
Update staging #3878

### CODE REVIEW

File: source/developers/projects/studio/api/publish/reset-staging.rst [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 .. _crafter-studio-api-publish-reset-staging:
 
-=========================
-Reset Staging Environment
-=========================
+=============
+Reset Staging
```
Comment (sumerjabri) [inline source/developers/projects/studio/api/publish/reset-staging.rst:6]: Change to `Staging Publishing Target`

File: source/developers/projects/studio/api/publish/reset-staging.rst [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+=============
 
-Reset Crafter Studio's staging environment repository with live repository.
+Reset Crafter Studio's staging repository with live repository.
```
Comment (sumerjabri) [inline source/developers/projects/studio/api/publish/reset-staging.rst:9]: staging publishing target (remove the word repository also)

File: source/system-administrators/activities/authoring/staging.rst [hunk 3]
```diff
@@ -2,51 +2,49 @@
 
 .. _staging-env:
 
-===============================================
-Crafter CMS Intermediate Publishing Environment
-===============================================
+=========================================
+Configuring Publishing to Staging Preview
```
Comment (sumerjabri) [inline source/system-administrators/activities/authoring/staging.rst:6]: Staging Target

File: source/system-administrators/activities/authoring/staging.rst [hunk 4]
```diff
@@ -58,19 +56,25 @@ To view your site with respect to the staging environment, we need to set it up
...
+   studio.repo.published.staging: staging
 
-.. note::
+-------------------------
+Syncing Staging with Live
+-------------------------
+
+There are times when you need to sync the staging repository with the live repository. When adding the ``staging`` publishing target to an established site, the live repository is not cloned to the staging repository until an item has been published after setting up staging.  There may be some other scenarios, when you might need to sync the staging repository with the live repository.
+
+To sync the staging repository to the live repository, simply execute the ``reset-staging`` API.  See :ref:`crafter-studio-api-publish-reset-staging` for more details on the API
```
Comment (sumerjabri) [inline source/system-administrators/activities/authoring/staging.rst:79]: staging target with the live target

**Comment by russdanner** [comment]
Confirmeds

**Pull request merged**